build.sbt

```diff
@@ -4,7 +4,7 @@ crossScalaVersions := Seq("2.10.6", "2.11.7", "2.12.0-M3")
 val commonSettings: Seq[Setting[_]] = Seq(
   organization := "be.doeraene",
 
-  version := "0.1.1-SNAPSHOT",
+  version := "0.2.0-SNAPSHOT",
```

Comment:
I don't see any reason for bumping to 0.2.0, as everything stays backwards source and binary compatible. This change should be reverted, as well as those in `sbt-plugin-test/build.sbt` and `sbt-plugin-test/project/plugins.sbt`.